app/src/main/java/com/picketing/www/business/domain/Schedule.java

```diff
@@ -0,0 +1,13 @@
+package com.picketing.www.business.domain;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class Schedule {
+    private Long concertId;
```

Comment:
예진님 작업한 내용을 보면 공연이라는 도메인은 `Show` 로 표현되고있습니다
`showId`가 더 적절하지 않을까요?
```suggestion
    private Long showId;
```